nucleus/utils.py

```diff
@@ -144,18 +144,18 @@ def upload_to_presigned_url(presigned_url: str, file_pointer: IO):
 
 def serialize_and_write_to_presigned_url(
     upload_units: Sequence[Union["DatasetItem", Annotation]],
-    dataset_id: str,
+    request_id: str,
     client,
 ):
-    request_id = uuid.uuid4().hex
+    request_uuid = uuid.uuid4().hex
     response = client.make_request(
         payload={},
-        route=f"dataset/{dataset_id}/signedUrl/{request_id}",
+        route=f"dataset/{request_id}/signedUrl/{request_uuid}",
```

Comment:
If this path is going to be able to accept ids that aren't just dataset ids, we need to change the routing. Maybe something like `presignRequest/request_id` or something.

Reply:
I don't want to break old clients though hmmm